cpp/tests/copying/concatenate_tests.cu

```diff
@@ -326,6 +326,9 @@ TEST_F(ListsColumnTest, ConcatenateEmptyLists)
 {
   // to disambiguiate between {} == 0 and {} == List{0}
   using LCW = cudf::test::lists_column_wrapper<int>;
+// clang-format off
+  #define EL LCW{}
+  // clang-format on
```

Comment:
What is this for?

Reply:
The "EL" reads cleaner for "empty list" than "LCW{}".  I retrofitted everything in the various tests to try and set this standard now.

As for clang formatting - it was mangling those #defines in some cases (but only some!), adding a \ and a newline which as causing compile errors.  Not really sure why.

Reply:
Don't use a macro like this. It makes it pretty inscrutable. If you want a special type for an empty list, just make a new type that represents an empty list column wrapper.